net-libs/neon/neon-0.34.0.ebuild

```diff
@@ -0,0 +1,105 @@
+# Copyright 2001-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools multilib-minimal
+
+DESCRIPTION="HTTP and WebDAV client library"
+HOMEPAGE="https://notroj.github.io/neon/ https://github.com/notroj/neon"
+SRC_URI="https://notroj.github.io/neon/${P}.tar.gz"
+
+LICENSE="GPL-2+"
```

Comment:
Why did you change license from GPL-2 to GPL-2+? I found https://github.com/notroj/neon/blob/0.34.0/src/COPYING.LIB and https://github.com/notroj/neon/blob/0.34.0/test/COPYING, booth look GPL-2.

Reply:
Ok, I see, thanks.

It is always better to give links for such findings beforehand as it saves review time, provides clear justification upfront, and avoids unnecessary back-and-forth discussions.

Reply:
Yes. Please include this sort of thing in the commit message in future.